<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="cbxVideoResolutions"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="edWidth">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="value">
            <number>1920</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Height:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="edHeight">
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="value">
            <number>1080</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cbxKeepAspectRatio">
         <property name="text">
          <string>Keep aspect ratio</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="ffmpeg">
           <property name="text">
            <string>ffmpeg.exe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFindFFmpeg">
           <property name="text">
            <string>Find FFmpeg</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="Codec"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblDuration">
           <property name="text">
            <string>Duration [sec.]:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Duration">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Framerate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Framerate">
           <property name="minimum">
            <number>24</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
